src/policy/fees.h

```diff
@@ -225,19 +225,19 @@ class CBlockPolicyEstimator
     void processBlock(unsigned int nBlockHeight,
                       std::vector<CTxMemPoolEntry>& entries, bool fCurrentEstimate);
 
-    /** Process a transaction confirmed in a block*/
+    /** Process a transaction confirmed in a block */
```

Comment:
I don't think CommentPragmas support this. They are a [regular expression that describes comments with special meaning, which should not be split into lines or otherwise changed. ](http://clang.llvm.org/doxygen/structclang_1_1format_1_1FormatStyle.html#ab66e80203947743add360933aa7a2256)